lib/src/__tests__/DateTimePicker.test.tsx

```diff
@@ -83,7 +80,44 @@ describe('e2e - DateTimePicker', () => {
         .at(0)
         .text()
     ).toBe('2018');
-    // expect(component.find('ToolbarButton').at(1).text()).toBe('Jan 3');
+  });
+
+  it('prop: minDateTime – hours is disabled by date part', () => {
+    const { getByLabelText } = render(
+      <DesktopDateTimePicker
+        open
+        openTo="hours"
+        dateAdapter={new DateFnsUtils()}
+        renderInput={props => <TextField autoFocus {...props} />}
+        mask="__.__.____ __:__"
+        onChange={() => {}}
+        value={utilsToUse.date('2018-01-01T00:00:00.000Z')}
+        minDateTime={utilsToUse.date('2018-01-01T12:30:00.000Z')}
+      />,
+      { strict: false }
+    );
+
+    expect(getByLabelText('11 hours').getAttribute('aria-disabled')).toBe("true");
```

Comment:
You might prefer, at least, I believe @eps1lon would recommend it to improve the failing message (if false)

```suggestion
    expect(getByLabelText('11 hours)).to.have.attribute('aria-disabled', 'true');
```

Reply:
We do not have chai installed, I was thinking about adding https://github.com/testing-library/jest-dom